Controller/FolderController.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * File containing the FolderController class.
```

Comment:
We no longer add the class name in the header (it's already there in the class anyway). Use "This file is part of the ezsystems/demo-bundle package".